package.json

```diff
@@ -48,7 +51,8 @@
     "jest": "26.6.3"
   },
   "resolutions": {
-    "@changesets/git": "1.1.1"
+    "@changesets/git": "1.1.1",
+    "prettier": "2.3.0"
```

Comment:
Why do we need to downgrade the dependency here?

Reply:
that's not a downgrade, 2.3.0 is the latest, and it's because another dependency is requesting an old 1.x version of prettier

Reply:
Oops, I compared it to `3.1.0` of `pretty-quick` :D thanks @PabloSzx